test/old-api/env.js

```diff
@@ -215,6 +215,7 @@ describe("jsdom/env", () => {
   specify("with configurable resource loader", { async: true }, t => {
     env({
       html: "<!DOCTYPE html><html><head><script src='foo.js'></script></head><body></body></html>",
+      url: "http://example.org/",
```

Comment:
What does this fix?

Reply:
url default is about:blank, so foo.js can't be resolved, so the custom resource loader isn't executed.

Reply:
How was this working before? :-/

Reply:
🌠 resource loader!

Reply:
But, what changed?

Is this in anticipation of other changes you have locally or something, that will break resolving `foo.js` against `about:blank`?

Reply:
These were all changes which I initially developed for streaming with parse5. Just trying to take what I can get from that PR and make it easier in the future, then close it out and maybe restart it down the road again from mostly scratch.